vobject/hcalendar.py

```diff
@@ -72,59 +72,64 @@ def out(s):
             # SUMMARY
             summary = event.getChildValue("summary")
             if summary:
-                out('<span class="summary">' + summary + '</span>:' + CRLF)
+                out('<span class="summary">' + summary + "</span>:" + CRLF)
 
             # DTSTART
             dtstart = event.getChildValue("dtstart")
             if dtstart:
-                if type(dtstart) == date:
+                if type(dtstart) is date:
```

Comment:
I replaced these with
    isinstance(dtstart, date)

which I like better, because it allows the use of a derived type if you want/need to do that for some reason.

Reply:
[class datetime(date)](https://github.com/py-vobject/vobject/commit/7ad25bb4fb9188548ba4443ff840b19b7187b0f8)

Due to this reason, `type` is a simpler way as it doesn't depend on the order of `if/else`. Using `isinstance` will require reordering the code in all conditions and make sure it stays likewise in future. (maintainability issue)

Ideally, there shouldn't be so many type/instance checks. So, I kept the logic as it is (to refactor later) and cleared linting error only.